app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: :User
+  belongs_to :posts
+end
```

Comment:
- [ ] Kindly add a method in Like model  that updates that updates the likes counter for a post.

eg:

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```